ibm_mq/datadog_checks/ibm_mq/connection.py

```diff
@@ -37,11 +43,12 @@ def get_normal_connection(config):
         queue_manager.connect_with_options(config.queue_manager_name, **kwargs)
     else:
         log.debug("connecting without a username and password")
-        queue_manager.connect_with_options(config.queue_manager, channel_definition)
+        queue_manager.connect_with_options(config.queue_manager_name, channel_definition)
     return queue_manager
 
 
 def get_ssl_connection(config):
+    # type: (IBMMQConfig) -> QueueManager
     """
```

Comment:
Maybe we can add types to `_get_channel_definition` too ?

Reply:
Oh, yeah, that was added on a separate PR